index.html

```diff
@@ -14,10 +14,31 @@
     <link rel="stylesheet" href="about.css">
     <link rel="stylesheet" href="contact.css">
     <link rel="stylesheet" href="desktop.css">
+    <link rel="stylesheet" href="menu.css">
+    <script defer src="script.js"></script>
     <title>Portfolio</title>
 </head>
 <body>
-    <header>
+
+    <div id="menu">
+    </div>
+
+    <div id="menu-items">
+        <div class="close-btn"><button id="close-menu"><img src="images/Cancel.svg" alt="cancel-button"></button></div>
+        <nav class="menu-items-child">
+            <div class="menu-frame"><a onclick="closeDisplay()" href="#portfolio">
+                    <p>Portfolio</p>
+                </a></div>
+            <div class="menu-frame"><a onclick="closeDisplay()" href="#about">
+                    <p>About</p>
+                </a></div>
+            <div class="menu-frame"><a onclick="closeDisplay()" href="#contact">
+                    <p>Contact</p>
+                </a></div>
+        </nav>
+    </div>

```

Comment:
- [ ] Kindly make sure when the mobile menu is open to remove scrolling, look at the following image to understand what I mean.
> ![image](https://user-images.githubusercontent.com/53402600/224799500-fcf9ed49-e206-4ca4-aa29-cb88242fac92.png)

Reply:
Okay I have fixed the issue thank you so much.